labels-for-feature-icons

mamuna36 opened the pull request:
added labels for feature icons

mamuna36 committed:
label for mirror added

mamuna36 committed:
label for mirror added

mamuna36 committed:
changes made

cunning-weasel committed:
Merge branch 'feature-labels' of github.com:FBW41-2/toilet-app-live
pulling in latest version

cunning-weasel committed:
grabing latest master

cunning-weasel committed:
grabing latest master

cunning-weasel committed:
fixing strange layout

galymax committed:
fixed layout issues

mamuna36 committed:
labels added

mamuna36 committed:
merge problem

mamuna36 committed:
label added

mamuna36 committed:
label amended

mamuna36 committed:
style changed

mamuna36 committed:
zip code field corrected

mamuna36 committed:
grid-solution

mamuna36 committed:
almost there

mamuna36 committed:
labels and hover effect for feature-icons

mamuna36 committed:
hover effect removed

galymax commented:
I would suggest to flip the labels with the icons, so the icons are on the left, because I think it will be easier to understand and it will adhere to the grid better, right now its moved to the right, which looks strange.

As you can see, your changes will create conflicts when merging into master which indicates that you didn't merge master into your branch before pushing your branch. Please do that and push again.

How come you removed the hover effect from the buttons, that another team created? Did you discuss that with anyone?

mamuna36 commented:
Hi Maxim,
I can flip the icons, I added hover effects myself and then found out another team has done it so I removed the hover effect in my version..I guess so, may be I made a mistake. After everything I recheck the version in the master branch and the hover effect was working. May be we can discuss this.



________________________________
From: Maxim <notifications@github.com>
Sent: Monday, September 21, 2020 1:36 PM
To: FBW41-2/toilet-app-live <toilet-app-live@noreply.github.com>
Cc: mamuna36 <mamuna_anwar@hotmail.com>; Author <author@noreply.github.com>
Subject: Re: [FBW41-2/toilet-app-live] Feature2 (#34)


I would suggest to flip the labels with the icons, so the icons are on the left, because I think it will be easier to understand and it will adhere to the grid better, right now its moved to the right, which looks strange.

As you can see, your changes will create conflicts when merging into master which indicates that you didn't merge master into your branch before pushing your branch. Please do that and push again.

How come you removed the hover effect from the buttons, that another team created? Did you discuss that with anyone?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/FBW41-2/toilet-app-live/pull/34#issuecomment-696059300>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AQQJSQME3B3MVMZV6MATZWLSG43DRANCNFSM4RQ6Z2TQ>.

mamuna36 committed:
icons and labels fliped

mamuna36 committed:
merge conflict resolved

galymax committed:
solved merge conflict in index.html

cunning-weasel committed:
added spacing to icons

cunning-weasel committed:
increased font size to match the rest of the doc

cunning-weasel committed:
shifted features text closer to icons

cunning-weasel committed:
added cursor pointer effect for icons + text

cunning-weasel committed:
last touches on icon text

galymax commented:
The space between the icon and the label is still too big
![image](https://user-images.githubusercontent.com/1148423/93869326-f2c85f80-fccb-11ea-9aa4-9144f0a112c0.png)

mamuna36 committed:
label for feature icons corrected

mamuna36 committed:
merge conflict resolved

mamuna36 committed:
files renamed to match the master branch

mamuna36 committed:
files renamed to match the master branch

mamuna36 committed:
merged with master and conflict resolved

mamuna36 commented:
I have reduced the spacing between the icons and labels.
